Sources/MapboxNavigation/ArrivalController.swift

```diff
@@ -51,11 +46,10 @@ class ArrivalController: NavigationComponentDelegate {
         
         embedEndOfRoute(into: viewController, onDismiss: onDismiss)
         endOfRouteViewController.destination = destination
-        navigationViewData.navigationView.endOfRouteView?.isHidden = false
```

Comment:
I think we should keep the constraint because it only aligns the bottom anchor of `endOfRouteView` to the bottom anchor of the `NavigationView`. Instead, we should adjust the `EndOfRouteViewController` design.